manager/state/raft/raft.go

```diff
@@ -115,6 +122,8 @@ type Node struct {
 	stopMu sync.RWMutex
 	// used for membership management checks
 	membershipLock sync.Mutex
+	// used to set Node
+	nodeLock sync.Mutex
```

Comment:
Race detector still complains as we set `etcd.raft.Node` after calling NewNode and returning a `raft.Node` object I guess.